apps/speed.c

```diff
@@ -3397,6 +3412,13 @@ int speed_main(int argc, char **argv)
                 EVP_PKEY_free(ed_pkey);
                 break;
             }
+	    if (!EVP_DigestVerifyInit(loopargs[i].eddsa_ctx2[testnum], NULL,
```

Comment:
Indentation seems to be off here

Reply:
I have to wonder if this ever worked....

Reply:
So git bisect says here:
> good: [8a5cb59601fa9892e22e26337917cf513d57c473] TEST: Add a test of keygen with >an empty template in test/evp_extra_test.c
> git bisect good 8a5cb59601fa9892e22e26337917cf513d57c473
> bad: [c1e48c5171affd99111fd08e7a0a7e1a76121138] s390: ECX key generation fixes.
> git bisect bad c1e48c5171affd99111fd08e7a0a7e1a76121138
> bad: [43cd37014ef4433ae8e82ba64bddc42cf0bd618a] ecx: add key generation support.
> git bisect bad 43cd37014ef4433ae8e82ba64bddc42cf0bd618a
> skip: [1ee1e551148d963b566a69c2e6e9a583646112fc] Add ECX to FIPS provider as >non-FIPS algorithms
> git bisect skip 1ee1e551148d963b566a69c2e6e9a583646112fc
> only skipped commits left to test
> possible first bad commit: [43cd37014ef4433ae8e82ba64bddc42cf0bd618a] ecx: add key >generation support.
> possible first bad commit: [1ee1e551148d963b566a69c2e6e9a583646112fc] Add ECX to >FIPS provider as non-FIPS algorithms
> c1e48c5171aff s390: ECX key generation fixes.
> 43cd37014ef44 (refs/bisect/bad) ecx: add key generation support.
> 1ee1e551148d9 (refs/bisect/skip-1ee1e551148d963b566a69c2e6e9a583646112fc) Add ECX >to FIPS provider as non-FIPS algorithms
> 8a5cb59601fa9 (refs/bisect/good-8a5cb59601fa9892e22e26337917cf513d57c473) TEST: >Add a test of keygen with an empty template in test/evp_extra_test.c

Would you consider this as a regression then?

Reply:
Well, this dubious code seems to have been there since it was first committed in d3a9fb10e...but I've not actually tested to see if it *did* actually somehow work then.

Reply:
Well, according to git bisect either 43cd370 or 1ee1e55 (which could not be tested) led to change in behavior using the same ctx for sign & verify. Since then it is either or.

Reply:
Well...either way, its still a bug that should be fixed. Please fix the indentation here, and otherwise this PR looks good.